corehq/apps/app_manager/templates/app_manager/partials/settings/app_settings.html

```diff
@@ -187,7 +187,11 @@ <h6>Example:</h6>
             </p>
             <a class="btn btn-primary"
                href="{% url 'pull_master_app' domain app.id %}">
-                {% trans 'Update from Master App' %}
+                {% trans 'Update from Master App now' %}
+            </a>
+            <a class="btn btn-primary"
+               href="{% url 'pull_master_app' domain app.id %}?notify=true">
```

Comment:
Since using real booleans doesn't work cleanly here, I think it's more readable to use `notify=1` rather than the string `'true'`.

Reply:
> Since using real booleans doesn't work cleanly here

Can you share the reason? Ultimately in the view we always would need to string comparison. Or does it look better in urls?

Reply:
Nevermind, I don't think it really matters. I tend to forget some of javascript's rules around string => boolean conversions, but thinking about it some more, I don't think "1" is really any better than "true".